### PR TITLE
Create 0000-01-02-r4ty34r.md

### PR DESCRIPTION
**Pull request by ZeroExcodes**
first.time_ey

### CODE REVIEW

File: _posts/_posts/0000-01-02-r4ty34r.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```
Comment (r4ty34r) [inline _posts/_posts/0000-01-02-r4ty34r.md:5]: #techbroloading

**Review by ZeroExcodes — COMMENTED** [review]
nice

File: _posts/_posts/0000-01-02-r4ty34r.md (hunk 1 above)
Comment (ZeroExcodes) [inline _posts/_posts/0000-01-02-r4ty34r.md]: #ok

Comment (ZeroExcodes) [inline _posts/_posts/0000-01-02-r4ty34r.md]: + get to the bag, never settle cant stop wont stop gamestop

Comment (ZeroExcodes) [inline _posts/_posts/0000-01-02-r4ty34r.md]: get to it gz

**Pull request merged**